src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,21 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+export type TunnelTimeDimension = 'access_key' | 'country' | 'asn';
+
+interface TunnelTimeRequest {
+  hours: number;
+}
+
+interface TunnelTimeResponse {
+  location?: string;
+  asn?: number;
+  tunnel_time_seconds: number;
```

Comment:
This is inconsistent with the way we do time in other places in the API. We use something like:

```ts
{
  tunnel_time: {
    seconds: number
  }
}
```

Reply:
Ugh I didn't realize we already use this format elsewhere. I suppose we could continue using it for consistency's sake, but it's an odd pattern to represent time in an API.

At least `seconds` is better than `hours`, so I guess that's something? It gives us the option to add nanoseconds in future and be more similar to the [`Timestamp` proto](https://github.com/protocolbuffers/protobuf/blob/6b22fd125ce20f56e89d0f207e36383d19a103fd/src/google/protobuf/timestamp.proto#L133). Let's not use it with anything other than `seconds` though, and we can consider fixing this in some unknown future.

Reply:
Note that this is not a timestamp. This is a duration. I could imagine a Duration type that can have whatever unit we want, and a function that goes from JSON to a Duration object. (I think we already do that?)